### PR TITLE
add doc for laravel support

### PR DESCRIPTION
**Pull request by KannarFr**
(laravel doesn't need monolog but only a simple env var to send logs to syslog).

### CODE REVIEW

File: runtimes/php/php-apps.md [hunk 1]
```diff
@@ -250,26 +250,12 @@ You can find more documentation about composer configuration at [getcomposer.com
 
 You use Artisan to manage your project and you want to execute _artisan migrate_ before running your app.
 
-First, add a file `ccbuild.sh` at the root of your project with these lines:
+To do this, we use a post build hook, you have to set a new environment variable on your Clever application as following
```
Comment (urcadox) [inline runtimes/php/php-apps.md]: Can you do this in another commit?

File: runtimes/php/php-apps.md [hunk 2]
```diff
@@ -446,6 +436,16 @@ monolog:
...
 
+Laravel doesn't need Monolog to retrieve logs via Clever console or Clever CLI. Here, ensure that you have the following line in `config/app.php`:
+
+```php
+...
+'log' => env('APP_LOG', 'syslog'),
+...
+```
+
+Then, set `APP_LOG=syslog` as Clever application environment variable.
```
Comment (urcadox) [inline runtimes/php/php-apps.md]: Huh, is that really necessary? I would guess that you need to either use the configuration file which sets the env variable OR define the env variable yourself.

Comment (KannarFr) [inline runtimes/php/php-apps.md]: I used the official getting started of laravel and IIRC that's the most simple way to configure logs.

File: runtimes/php/php-apps.md (hunk 1 above)
Comment (urcadox) [inline runtimes/php/php-apps.md]: @KannarFr Can you fix this issue and then I can merge this. Feel free to add another commit which reverts those changes and then I will squash & merge.
